include/ppx/application.h

```diff
@@ -476,6 +479,9 @@ class Application
     uint32_t    mStereoscopicSwapchainIndex = 0;
     ImVec2      lastImGuiWindowSize         = {};
 #endif
+
+protected:
+    KnobManager mKnobManager = {};
```

Comment:
KnobManager have a constructor, so `KnobManager mKnobManager;` is enough for initialization.

Also make it private, and have a protected `GetKnobManager()`, or just pass it to `InitKnobs` as `InitKnobs(KnobManager&)`

Reply:
It's more of a google c++ style preference to have all data member private. So

```
class Application {
protected:
  KnobManager& GetKnobManager() {return mKnobManager;}
private:
  KnobManager mKnobManager;
};
... ProjApp::UpdateGUI() {
  ...
  GetKnobManager().DrawAllKnobs();
  ...
}
```